extension/src/test/fixtures/expShow/base/customPlots.ts

```diff
@@ -178,18 +248,18 @@ const data: CustomPlotsData = {
       metric: 'summary.json:loss',
       param: CHECKPOINTS_PARAM,
       values: [
-        { group: 'exp-83425', iteration: 1, y: 1.9896177053451538 },
-        { group: 'exp-83425', iteration: 2, y: 1.9329891204833984 },
-        { group: 'exp-83425', iteration: 3, y: 1.8798457384109497 },
-        { group: 'exp-83425', iteration: 4, y: 1.8261293172836304 },
-        { group: 'exp-83425', iteration: 5, y: 1.775016188621521 },
-        { group: 'exp-83425', iteration: 6, y: 1.775016188621521 },
-        { group: 'test-branch', iteration: 1, y: 1.9882521629333496 },
-        { group: 'test-branch', iteration: 2, y: 1.9293040037155151 },
-        { group: 'test-branch', iteration: 3, y: 1.9293040037155151 },
-        { group: 'exp-e7a67', iteration: 1, y: 2.020392894744873 },
+        { group: 'exp-e7a67', iteration: 3, y: 2.0205044746398926 },
```

Comment:
Due to the way `experimentWithCheckpoints` is formatted compared to `ExperimentOutput`, our checkpoint values are kind of "backwards" :) From my understanding, this change won't affect the rendered plots, but please correct me if I'm wrong!